src/main/webapp/script.js

```diff
@@ -64,26 +64,26 @@ function loadSortValue() {
  */
 function getDashboardItems(clearCurrentItems = true, cursor = '') {
   fetch(
-      '/dashboard?cursor=' + cursor + '&searchValue=' +
-      $('#searchValue').val() + '&sortType=' + $('#sortType').val())
-      .then((response) => response.json())
-      .then((entertainmentItemList) => {
```

Comment:
Is it possible if you can remove this file from the PR since it seems like all it does is change the formatting? I am making lots of changes in this file for my current PR and I think this could cause a lot of merge conflicts. Also is this change in format the same as the one from clang-format?